_includes/head.html

```diff
@@ -12,6 +12,9 @@
   <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ "/feed.xml" | prepend: site.baseurl | prepend: site.url }}" />
 
+  <link rel="shortcut icon" href="{{ "/images/favicon.ico" | prepend: site.baseurl }}" type="image/x-icon">
+  <link rel="icon" href="{{ "/images/favicon.ico" | prepend: site.baseurl }}" type="image/x-icon">
```

Comment:
That's a great question.  I did some research on favicons and there is an obnoxious variety of ways you can import/link these things.  It reminds me of vendor-prefixed css...  :cry:   (see http://realfavicongenerator.net/ as an example).  Maybe the thing to do is to just use:
```
<link rel="icon"
```
and then layer on others if/when the need arises.